appveyor.yml

```diff
@@ -14,14 +14,21 @@ environment:
   matrix:
   - APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu1804
     LANG: us_US.UTF-8
-  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+  - APPVEYOR_BUILD_WORKER_IMAGE: Previous Visual Studio 2017

```

Comment:
It would be nice to have a comment here with a link to the GitHub issue that is causing us to have to use the `Previous` image.